nbi/buildspec.yml

```diff
@@ -0,0 +1,55 @@
+version: 0.2
+env:
+  shell: bash
+  variables: {}
+phases:
+  install:
+    commands:
+      - _ANACONDA_ARCHIVE_URL=https://repo.anaconda.com/archive
+      - _MINICONDA_ARCHIVE_URL=https://repo.anaconda.com/miniconda
+      - export LCC_CONDA_INSTALLER=Anaconda3-2020.07-Linux-x86_64.sh
```

Comment:
We could also consider using the latest conda installer: [Anaconda3-2022.10-Linux-x86_64.sh](https://docs.anaconda.com/anaconda/install/hashes/Anaconda3-2022.10-Linux-x86_64.sh-hash/)

Reply:
per discussion, this comment and the next are out of scope